cpython/Makefile

```diff
@@ -133,6 +133,7 @@ $(BUILD)/Makefile: $(BUILD)/.patched $(ZLIBBUILD)/.configured $(INSTALL)/lib/lib
 			  --without-pymalloc \
 			  --disable-shared \
 			  --disable-ipv6 \
+			  --enable-big-digits=30 \
```

Comment:
Would be worth adding a comment why we want this.

Reply:
Sounds good. 

The reason is as follows: there are two options for this, 15 and 30. 30 is faster as long as the system has good support for 64 bit arithmetic, otherwise 15 is faster. By default Python uses 15 if the system has 32 bit pointers and 30 if the system has 64 bit pointers. The majority of users are running 64 bit systems and even most systems with 32 bit pointers have efficient 64 bit arithmetic so I think for most machines arithmetic will be faster with this set to 30.

Reply:
Yeah, but the WASM VM is 32 bit, so I'm not sure we would really see the impact of 64bit systems?

Reply:
It has 32 bit pointers but it has native 64 bit arithmetic.

Reply:
For instance the following
```C
long long test(long long a, long long b){
    return a * b;
} 
```
compile to native 64 bit arithmetic instructions:
```wat
(func $test (type 0) (param i64 i64) (result i64)
    (local i32 i32 i32 i64 i64 i64)
    ;; omitted preamble
    i64.load offset=8
    local.set 5
    local.get 4
    i64.load
    local.set 6
    local.get 5
    local.get 6
    i64.mul
    ;; omitted postamble
    return
)
```

Reply:
Yes, indeed, makes sense.